source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -307,6 +307,28 @@ requirements shown in Table 1:
      - /
      - *Size depends upon use case/desired bundles.*
 
+Disk encryption
+---------------
+
+Disk encryption is supported for root and swap partitions on |CL| for
+greater security. Encryption is optional. To encrypt a partition, see the example below in the :ref:`encrypt-root`.
+
+.. note::
+
+   **The `boot` partition cannot be encrypted**.
+
+Encryption Passphrase
+---------------------
+
+|CL| supports use of a single passphrase for all encrypted partitions.
+Therefore, if you turn on encryption for one partition, the same passphrase
+must be used for all partitions. For example, suppose you encrypt the `root`
+partition first and the `swap` partition second. Upon selecting encryption
+for the `swap` partition, the :guilabel:`Encryption Passphrase` dialogue
+appears showing the current default Passphrase. Select :guilabel:`Confirm`
+and press :kbd:`Enter` to use the same Passphrase.
+
+
```

Comment:
.. note::

    **Additional or different keys may be configured post-installation using `cryptsetup` tool.**.

Reply:
Resolved above. See next commit.